pom.xml

```diff
@@ -67,7 +67,7 @@
         <maven.shared-utils.version>3.2.1</maven.shared-utils.version>
         <maven.version>3.3.3</maven.version>
 
-        <azure.toolkit-lib.version>0.7.0</azure.toolkit-lib.version>
+        <azure.toolkit-lib.version>0.7.1</azure.toolkit-lib.version>
```

Comment:
this is not need. as maven plugin doesn't need this fix

Reply:
as discussed in the office with Andy and Hanxiao, upgrade them in a same version.
we can re-consider it again tomorrow if you have any concern.

Reply:
Remove change of this pom file.